assignment.js

```diff
@@ -32,6 +32,20 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let baseNumber = 2
+let i = 0
+
+do {
+  console.log(baseNumber * i)
+  baseNumber && i++
```

Comment:
@nurisellie Nice use of the `do...while` loop but it looks like this part of line 40: `baseNumber &&` is not needed (and in coding we want to avoid adding code that is not needed because), all you need to do is increment `i` on this line (which you're already doing with `i++`).